apps/ledger-live-mobile/src/screens/Onboarding/steps/pairNew.tsx

```diff
@@ -129,11 +129,13 @@ function OnboardingStepPairNew() {
       screen: NavigatorName.Main,
     });
 
-    startPostOnboarding(deviceModelId as DeviceModelId, false, () =>
-      navigation.navigate(NavigatorName.Base, {
-        screen: NavigatorName.Main,
-      }),
-    );
+    startPostOnboarding({
+      deviceModelId: deviceModelId as DeviceModelId,
+      fallbackIfNoAction: () =>
+        navigation.navigate(NavigatorName.Base, {
+          screen: NavigatorName.Main,
+        }),
+    });
```

Comment:
- this function's signature changed, now passing options in an object for more clarity
- also removing the "false" which is now the default value for `mock`